docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
+\section{Limitations}\label{sec:limits}
+There are a few limitations to the accuracy of the phasing methods within \texttt{pyuvdata}. For most users, these limitations are well below the threshold of significance for the data under analysis, but those doing very high resolution or astrometry measurements following the use of these methods should take heed of the information below.
 
-We extracted a single time and 10 frequencies from these files and used the cut-down files in unit tests (in \verb!test_uvdata.py!).
+\subsection{Precision Limits}
+Accuracy of the astrometry is only guaranteed to $\sim1$ mas, tested at the highest precision level against data sets collected by the Submillimeter Array (SMA), which is generally sensitive to errors down to $\sim100$ mas. The level of precision in the results is guaranteed to be $\leq1$ mas, and in practice is generally closer to $\lesssim100$ {\textmu}as. This level of accuracy and precision is generally sufficient for all but the highest resolution interferometers, although users should be aware that these methods are not sufficiently accurate for VLBI-type phasing calculations (both for accuracy issues as well as some other effects discussed below). 
 
-\section{Future directions}
-The VLBI \texttt{calc} program is probably the most accurate code available for comparing phasing against. It is possible that it  includes some effects that we don't wish to include, however. The information we have about \texttt{calc} comes from the following email from Randall Wayth on other ways to test phasing calculations:
-\begin{quotation}
-Way back when we were trying to sort out the precession corrections for MWA, I was comparing the output from Al's new code to that of the VLBI program called `calc'. `calc' is the tool that most (all?) VLBI observatories use to calculate delays and u, v, w coords for data, and as far as I know is a fully complete system with all known effects. For low frequencies it is overkill of course, but as a reference system it is probably ideal.
+\subsection{Compensation for Decoherence}
+Phasing in \texttt{pyuvdata} works by applying a frequency-dependent phase correction to the data to compensate for the differential delay between antennas. The phase correction is calculated at the center of the channel, but when phasing well away from the correlator phase center, decoherence is introduced due to the change in phase from one edge to another of a single frequency channel (sometimes called ``correlator squint'' or ``bandwidth smearing'') that \texttt{pyuvdata} does not compensating for. The exact impact depends on the details of the instrument, but the magnitude can be estimated as $\sim\Delta w \cdot \delta\nu / c$, where $\Delta w$ is the change in $w$-position of a given baseline, $\delta\nu$ is the channel width, and $c$ is the speed of light.
```

Comment:
that \texttt{pyuvdata} does not compensating for -> that \texttt{pyuvdata} does not compensate for

Reply:
Fixed.